lib/dataContract/DataContract.js

```diff
@@ -226,19 +249,19 @@ class DataContract {
   /**
    * Set Data Contract entropy
    *
-   * @param {string} entropy
+   * @param {Buffer} entropy
```

Comment:
EncodedBuffer|Buffer|string

Reply:
Why? Last week you told me to change {string|buffer} to {buffer}